comixed-frontend/src/assets/i18n/es/library.json

```diff
@@ -2,484 +2,485 @@
+        "detail": "Error al bloquear los hashes de páginas."
       }
     },
     "set-deleted": {
       "success": {
-        "detail": "Successfully {deleted, select, true{marked} other{unmarked}} selected pages."
+        "detail": "Páginas seleccionadas {deleted, select, true{marcadas} other{desmarcadas}} correctamente."
       },
       "error": {
-        "detail": "Failed to {deleted, select, true{mark} other{unmark}} pages for deletion."
+        "detail": "Error al {deleted, select, true{marcar} other{desmarcar}} páginas para eliminación."
       }
     }
   },
   "duplicates-page-toolbar": {
     "options": {
       "blocking": {
-        "turn-on": "Block",
-        "turn-off": "Unblock"
+        "turn-on": "'Bloquear'",
+        "turn-off": "'Desbloquear'"
       },
       "deleting": {
-        "mark": "Mark",
-        "unmark": "Unmark"
+        "mark": "Marcar",
+        "unmark": "Desmarcar"
       }
     },
     "button": {
-      "blocking-options": "Blocking",
-      "deleting-options": "Deleting"
+      "blocking-options": "Bloqueando*****",
+      "deleting-options": "Eliminando*****"
     },
     "set-blocking": {
-      "header": "Set Blocking State For Pages",
-      "message": "Are you sure you want to {blocking, select, true{block} other{unblock}} the selected pages?"
+      "header": "Establecer estado de 'bloqueo' para las páginas",
+      "message": "¿Seguro que quieres {blocking, select, true{'bloquear'} other{'desbloquear'}} las páginas selecciondas?"
     },
     "set-deleted": {
-      "header": "Mark Pages For Deletion",
-      "message": "Are you sure you want to {deleted, select, true{mark} other{unmark}} the selected pages?"
+      "header": "Marcar páginas para eliminación",
+      "message": "¿Seguro que quieres {deleted, select, true{marcar other{desmarcar}} las páginas seleccionadas?"
     }
   },
   "duplicate-page-grid-item": {
-    "tooltip": "This page is {blocked, select, true{blocked} other{not blocked}}, and {deletedCount, plural, =0{none are} =1{# is} other{# are}} marked for deletion.",
-    "caption": "Found {count, plural, =1{one copy} other{# copies}}."
+    "tooltip": "Esta página está {blocked, select, true{'bloqueada'} other{no 'bloqueada'}}, and {deletedCount, plural, =0{ninguna está marcada} =1{# está marcada} other{# están marcadas}} para eliminación.",
+    "caption": "Encontrada{count, plural, =1{una copia} other{s # copias}}."
   },
   "duplicate-page-list-item": {
-    "title": "Found {count, plural, =1{one copy} other{# copies}}.",
-    "subtitle": "This page is {blocked, select, true{blocked} other{not blocked}}, and {deletedCount, plural, =0{none are} =1{# is} other{# are}} marked for deletion."
+    "title": "Encontrada{count, plural, =1{una copia} other{s # copias}}.",
+    "subtitle": "Esta página está {blocked, select, true{'bloqueada'} other{no 'bloqueada'}}, and {deletedCount, plural, =0{ninguna está marcada} =1{# está marcada} other{# están marcadas}} para eliminación."
```

Comment:
Please remove the single quotes from around bloqueada in both spots here.

Reply:
Hopefully solved with commit [abd9ff7]